Automattic/wp-calypso
# Fix/jetpack plugins remove component

This PR removes the component "plugin-card-header", who by know were pretty much doing nothing but adding a <div>. 

It also updates `/client/my-sites/plugins/plugin-sections` to ES6 (because I couldn't commit the changes if I didn't do it before :D )

User-wise, there should be no visible change.

Before:
![image](https://cloud.githubusercontent.com/assets/1554855/18878210/12ef13e4-84d0-11e6-9658-57649035751f.png)


After:
![image](https://cloud.githubusercontent.com/assets/1554855/18878225/21c74666-84d0-11e6-877a-e9d61fc83044.png)



How to test
=========
Go to any plugin (you need to have at least a jetpack site connected to your account) and check the sections header is still there. For example, http://calypso.localhost:3000/plugins/jetpack
